### PR TITLE
fix: transition to new cmp

### PR DESCRIPTION
**Pull request by fkakatie**
## Description

transition to new cmp outlined in 2020 CMP Engineering Implementation

## Link

https://new-cmp--blog--adobe.hlx3.page/?helix-env=dev

### CODE REVIEW

File: scripts/delayed.js [hunk 1]
```diff
@@ -150,21 +150,19 @@ function loadPrivacy() {
...
   window.fedsConfig = {
+    locale: getLanguage(), // use supported locale
+    disableTarget: true, // this will bypass Target A/B tests
+    content: {
+      experience: 'privacy', // authored experience with Privacy component
+    },
     privacy: {
-      otDomainId: getOtDomainId(),
+      otDomainId: getOtDomainId(), // your OneTrust domain ID - see list of domains
+      footerLinkSelector: '[href="https://www.adobe.com/#openPrivacy"]', // CSS selector that will open the privacy modal
```
Comment (narcis-radu) [inline scripts/delayed.js:160]: this should work even if the DOM node is not available at the time privacy loads. Make sure to click on "cookie preferences" in the footer and test.

File: scripts/delayed.js [hunk 2]
```diff
@@ -150,21 +150,19 @@ function loadPrivacy() {
 
   // Configure Privacy
   window.fedsConfig = {
+    locale: getLanguage(), // use supported locale
```
Comment (narcis-radu) [inline scripts/delayed.js:153]: L#153 - L#157 are not needed when you load the standalone Privacy.

File: scripts/delayed.js [hunk 3]
```diff
@@ -150,21 +150,19 @@ function loadPrivacy() {
...
-  if (preferenceCenter) {
-    preferenceCenter.addEventListener('click', (event) => {
-      event.preventDefault();
-      window?.adobePrivacy?.showConsentPopup();
-    });
-  }
-
   const env = getHelixEnv().name === 'prod' ? '' : 'stage.';
-  loadScript(`https://www.${env}adobe.com/etc/beagle/public/globalnav/adobe-privacy/latest/privacy.min.js`);
+  loadScript(`https://www.${env}adobe.com/etc.clientlibs/globalnav/clientlibs/base/privacy.standalone.js`);
```
Comment (narcis-radu) [inline scripts/delayed.js:165]: I just sent an email with the updated URL - it's privacy-standalone.js (with a "-").

File: scripts/delayed.js [hunk 4]
```diff
@@ -150,21 +150,19 @@ function loadPrivacy() {
```
Comment (narcis-radu) [inline scripts/delayed.js:150]: the new PrivacyJS library does not need to be delayed as it won't load OneTrust every single time. Please make sure this "delayed.js" script is not actually delayed

**Review by narcis-radu — APPROVED** [review]
looks good to me, but it should not be merged until the new script gets published (February 3rd).

**Review by kptdobe — APPROVED** [review]

**Comment by narcis-radu** [comment]
hey @fkakatie - the script is now available on Production, you can test and merge this PR.
cc @kptdobe

**Pull request merged**